HCaptcha/Classes/HCaptchaWebViewManager.swift

```diff
@@ -176,6 +177,10 @@ internal class HCaptchaWebViewManager: NSObject {
 
         executeJS(command: .execute)
     }
+    
+    func reload() {
+        executeJS(command: .reload)
+    }
```

Comment:
The same point as above https://github.com/hCaptcha/HCaptcha-ios-sdk/pull/139/files#r1495004519